Changes

```diff
@@ -111,6 +111,9 @@ Working version
 - #9060: ensure that Misc.protect_refs preserves backtraces
   (Gabriel Scherer, review by Guillaume Munch-Maccagnoni and David Allsopp)
 
+- #9061: expose compiler Longident.t parsers
```

Comment:
Wrong number

Reply:
Thanks! Fixed in 09da7dd659b2c3c685cd1bd6f42aba9e87e42036.